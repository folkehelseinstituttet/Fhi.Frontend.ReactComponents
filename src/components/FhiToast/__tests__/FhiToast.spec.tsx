import renderer from 'react-test-renderer';
import Toast from '../FhiToast';

describe('FhiToast', () => {
  test('snapshot test', () => {
    const toast = renderer.create(<Toast id={1} text="This is a toast!" removeToast={() => {}} delay={0} />);
    const tree = toast.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
