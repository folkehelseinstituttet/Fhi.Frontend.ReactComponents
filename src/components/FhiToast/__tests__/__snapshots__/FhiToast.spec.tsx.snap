// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FhiToast snapshot test 1`] = `
<div
  aria-atomic="true"
  aria-live="assertive"
  className="fade toast FhiToast__MarginToast-sc-10dpyte-0 gCxjPv show"
  role="alert"
>
  <div
    className="FhiToast__ToastBody-sc-10dpyte-1 NfMJL toast-body"
  >
    This is a toast!
    <button
      className="FhiToast__CloseButton-sc-10dpyte-2 hARZvN btn-close ml-2 mb-1"
      data-dismiss="toast"
      type="button"
    >
      <span
        aria-hidden="true"
      >
        Ã—
      </span>
      <span
        className="sr-only"
      >
        Close
      </span>
    </button>
  </div>
</div>
`;
